static/app/gettingStartedDocs/javascript/javascript.tsx

```diff
@@ -111,43 +150,151 @@ const getInstallConfig = () => [
   },
 ];
 
-const onboarding: OnboardingConfig = {
-  introduction: MaybeBrowserProfilingBetaWarning,
-  install: () => [
-    {
-      type: StepType.INSTALL,
-      description: t(
-        'Sentry captures data by using an SDK within your application’s runtime.'
-      ),
-      configurations: getInstallConfig(),
-    },
-  ],
-  configure: (params: Params) => [
-    {
-      type: StepType.CONFIGURE,
-      description: t(
-        "Initialize Sentry as early as possible in your application's lifecycle."
-      ),
-      configurations: [
-        {
-          code: [
-            {
-              label: 'JavaScript',
-              value: 'javascript',
-              language: 'javascript',
-              code: getSdkSetupSnippet(params),
+const getLoaderInstallConfig = (params: Params) => [
+  {
+    language: 'html',
+    code: [
+      {
+        label: 'HTML',
+        value: 'html',
+        language: 'html',
+        code: `
+<script
+  src="${params.dsn.cdn}"
+  crossorigin="anonymous"
+></script>`,
+      },
+    ],
+  },
+];
+
+const getLoaderOptionalConfig = (params: Params) => [
+  {
+    language: 'html',
+    code: [
+      {
+        label: 'HTML',
+        value: 'html',
+        language: 'html',
+        code: `
+<script>
+  Sentry.onLoad(function() {
+    Sentry.init({${
+      !(params.isPerformanceSelected || params.isReplaySelected)
+        ? `
+        // You can add any additional configuration here`
+        : ''
+    }${
+      params.isPerformanceSelected
+        ? `
+        // Tracing
+        tracesSampleRate: 1.0, // Capture 100% of the transactions`
+        : ''
+    }${
+      params.isReplaySelected
+        ? `
+        // Session Replay
+        replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
+        replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.`
+        : ''
+    }
+      });
+  });
+</script>`,
+      },
+    ],
+  },
+];
+
+const onboarding: OnboardingConfig<PlatformOptions> = {
+  introduction: params => (
+    <div
+      css={css`
+        display: flex;
+        flex-direction: column;
+        gap: ${space(1)};
+      `}
+    >
+      <MaybeBrowserProfilingBetaWarning {...params} />
+      <TextBlock noMargin>
+        {isAutoInstall(params)
+          ? tct('In this quick guide you’ll use our [strong: Loader Script] to set up:', {
+              strong: <strong />,
+            })
+          : tct(
+              'In this quick guide you’ll use [strong:npm] or [strong:yarn] to set up:',
+              {
+                strong: <strong />,
+              }
+            )}
+      </TextBlock>
+    </div>
+  ),
+  install: params =>
+    isAutoInstall(params)
```

Comment:
I think we could improve a bit on the general structure here.
e.g. have two 3 configuration objects.
```ts
const loaderScriptOnboarding: {
  // ...
}

const packageManagerOnboarding: {
  // ...
}

const onboarding: {
  install:  isAutoInstall(params)
      ? loaderScriptOnboarding.install.(params)
      : packageManagerOnboarding.install.(params)
  // ...
}
```

Reply:
that is a good idea! I implemented that 🙂